src/core_init_atmosphere/mpas_init_atm_thompson_aerosols.F

```diff
@@ -0,0 +1,753 @@
+       elseif(index(field%field,'P_WIF_AUG') /= 0) then
+          k = field%xlvl
+          call mpas_log_write('Interpolating P_WIF_AUG at $i',intArgs=(/k/))
+          nInterpPoints = nCells
+          latPoints => latCell
+          lonPoints => lonCell
+          call mpas_pool_get_array(mesh,'pwif_gocart_clim',destField3d)
+          nmonths = 8
+       elseif(index(field%field,'P_WIF_SEP') /= 0) then
+          k = field%xlvl
+          call mpas_log_write('Interpolating P_WIF_SEP at $i',intArgs=(/k/))
+          nInterpPoints = nCells
+          latPoints => latCell
+          lonPoints => lonCell
+          call mpas_pool_get_array(mesh,'pwif_gocart_clim',destField3d)
+          nmonths = 9
+       elseif(index(field%field,'P_WIF_OCT') /= 0) then
+          k = field%xlvl
+          call mpas_log_write('Interpolating P_WIF_OCT at $i',intArgs=(/k/))
+          nInterpPoints = nCells
+          latPoints => latCell
+          lonPoints => lonCell
+          call mpas_pool_get_array(mesh,'pwif_gocart_clim',destField3d)
+          nmonths = 10
+       elseif(index(field%field,'P_WIF_NOV') /= 0) then
+          k = field%xlvl
+          call mpas_log_write('Interpolating P_WIF_NOV at $i',intArgs=(/k/))
+          nInterpPoints = nCells
+          latPoints => latCell
+          lonPoints => lonCell
+          call mpas_pool_get_array(mesh,'pwif_gocart_clim',destField3d)
+          nmonths = 11
+       elseif(index(field%field,'P_WIF_DEC') /= 0) then
+          k = field%xlvl
+          call mpas_log_write('Interpolating P_WIF_DEC at $i',intArgs=(/k/))
+          nInterpPoints = nCells
+          latPoints => latCell
+          lonPoints => lonCell
+          call mpas_pool_get_array(mesh,'pwif_gocart_clim',destField3d)
+          nmonths = 12
+       endif
+
+       allocate(rslab(-2:field%nx+3,field%ny))
+       rslab(1:field%nx,1:field%ny) = field%slab(1:field%nx,1:field%ny)
+       rslab(0,1:field%ny)  = field%slab(field%nx  ,1:field%ny)
+       rslab(-1,1:field%ny) = field%slab(field%nx-1,1:field%ny)
+       rslab(-2,1:field%ny) = field%slab(field%nx-2,1:field%ny)
+       rslab(field%nx+1,1:field%ny) = field%slab(1,1:field%ny)
+       rslab(field%nx+2,1:field%ny) = field%slab(2,1:field%ny)
+       rslab(field%nx+3,1:field%ny) = field%slab(3,1:field%ny)
+
+       do iCell = 1, nInterpPoints
+          if(mask_array(iCell) /= masked) then
+             lat = latPoints(iCell)*DEG_PER_RAD
+             lon = lonPoints(iCell)*DEG_PER_RAD
+             call latlon_to_ij(proj,lat,lon,x,y)
+             if(x < 0.5) then
+                lon = lon + 360.0
+                call latlon_to_ij(proj,lat,lon,x,y)
+             elseif(x > real(field%nx,kind=RKIND)+ 0.5) then
+                lon = lon - 360.0
+                call latlon_to_ij(proj,lat,lon,x,y)
+             endif
+
+             if(maskval /= -1.0) then
+                destField3d(nmonths,k,iCell) = interp_sequence(x,y,1,rslab,-2,field%nx+3,1,field%ny,1,1,msgval, \
```

Comment:
Compiling with the 'nvhpc' build target gives these errors:
```
NVFORTRAN-S-0023-Syntax error - unbalanced parentheses (mpas_init_atm_thompson_aerosols.F: 697)
NVFORTRAN-S-0023-Syntax error - unbalanced parentheses (mpas_init_atm_thompson_aerosols.F: 698)
NVFORTRAN-S-0023-Syntax error - unbalanced parentheses (mpas_init_atm_thompson_aerosols.F: 700)
NVFORTRAN-S-0023-Syntax error - unbalanced parentheses (mpas_init_atm_thompson_aerosols.F: 701)
```
I think converting the `\` to a `&` on these lines would resolve the issue.